designer/client/src/components/comment/StyledComment.tsx

```diff
@@ -0,0 +1,89 @@
+import { styled } from "@mui/material";
+import { NkButton } from "../NkButton";
+import { variables } from "../../stylesheets/variables";
+
+export const ProcessCommentsWrapper = styled("div")`
```

Comment:
Not sure if it's a good component for this comment, but there is a difference between bottom padding:

Localhost:

<img width="296" alt="image" src="https://github.com/TouK/nussknacker/assets/9945753/c5397d8b-5d0c-4354-bf4a-ecb98ae12eeb">

Staging:
<img width="289" alt="image" src="https://github.com/TouK/nussknacker/assets/9945753/ea7ee212-890b-4586-a0f7-352685f5fd1d">

Reply:
fix, and fix button size